doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -108,6 +108,44 @@ For each empty box in the matrix above, ensure that the following applications c
    gnome-terminal
    ```
 
+5. Test that `X11` apps can be started and can receive input when `XWayland` is supported:
+    ```sh
+    sudo apt install x11-apps
+    
+    # first,
+    xeyes
+
+    # then
+    xedit
+
+    # finally
+    xcalc
+    ```
+
+6. (X11) Test that `mattermost` can be opened and resized. This was first encountered in
+   [#2509](https://github.com/canonical/mir/issues/2509). To test:
+   
+   1. Run `miral-app` with a floating window manager
+   2. `sudo snap install mattermost-desktop`
+   3. `mattermost-desktop`
+   4. Resize mattermost
+   5. Confirm that the window is resized correctly
+
+7. (X11) You will need a CLion account for this one. Test that `clion` menus can be interacted
+   with. This was first encountered in [#2326](https://github.com/canonical/mir/issues/2326),
+   [#3107](https://github.com/canonical/mir/issues/3107),
+   [#3024](https://github.com/canonical/mir/issues/3024). 
```

Comment:
I think this was a longstanding bug, not a 2.16 regression. Also, this should be reproducible on any of the free, or free trial, JetBrains tools